src/secretDataType.tsx

```diff
@@ -0,0 +1,9 @@
+// Type Blog
+export interface SecretDataType {
```

Comment:
why is it `SecretDataType`? what's secret?

And you don't need `Type` in type names.